### PR TITLE
Runs JRuby tests on travis

### PR DESCRIPTION
**Pull request by pftg**
There were errors when tests were tried to be run on JRuby. Added fix and also enabled CI to run tests for jruby too.

### CODE REVIEW

**Comment by pftg** [comment]
@jcupitt could you check, the next PR will add truffleruby support

**Comment by jcupitt** [comment]
Hi @pftg, sorry, I wasn't sure if this was still a WIP. I'll have a look.

File: lib/vips/operation.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@ module Vips
                               GObject::GParamSpec.ptr,
                               ArgumentClass.ptr,
                               ArgumentInstance.ptr,
-                              :pointer, :pointer], :pointer
+                              :pointer, :pointer], :void
```
Comment (jcupitt) [inline lib/vips/operation.rb:27]: Could you explain why you need to remove this return pointer?

File: lib/vips/operation.rb [hunk 2]
```diff
@@ -66,7 +66,7 @@ def initialize name
           # names can include - as punctuation, but we always use _ in
           # Ruby
           arg_name = pspec[:name].tr("-", "_")
-          args << {
+          @args << {
```
Comment (jcupitt) [inline lib/vips/operation.rb:69]: Oops, good catch.

**Review by jcupitt — COMMENTED** [review]
All looks good, except for one small query.

**Comment by pftg** [comment]
@jcupitt thanks for notices. I removed not relevant changes, and instead of changing the scheme of the lib, just make return nil. 

Here is why:

For JRuby, it could not auto-convert array into a pointer. Because sometimes we return an array, this is not a pointer, then we need to convert array to pointer. But we do not use results anywhere, so better to return nil which will be auto-converted to the expected type.

**Comment by jcupitt** [comment]
OK, sounds reasonable.

Thank you for doing this work! I'll merge.

**Comment by jcupitt** [comment]
I noted this in CHANGELOG and credited you.

**Pull request merged**